Firebase/DynamicLinks/FIRDLJavaScriptExecutor.m

```diff
@@ -113,10 +113,11 @@ - (void)webView:(WKWebView *)webView
                 [webView
                     evaluateJavaScript:GINFingerprintJSMethodString()
                      completionHandler:^(id _Nullable result, NSError *_Nullable functionError) {
+                       __typeof__(self) strongSelf = weakSelf;
```

Comment:
Taking a closer look at the code, I'm not actually convinced `self` needs to be weak in either of these blocks. Removing the weak self declaration and using `self` instead looks like a simpler solution. @dmandar what do you think?

Reply:
Unless there is a reference cycle, there isn;'t an explicit need. It is fine to add this since we already had the weakself.